connectors/utils.py

```diff
@@ -234,6 +234,12 @@ async def _putter_timeout(self, putter):
     async def put(self, item):
         item_size = get_size(item)
 
+        if item_size > self.maxmemsize:
+            logger.debug(
```

Comment:
Can we use warning log as we should aware user about it that we are skipping this file?

Reply:
Makes sense, though with current level of indirection it's hard to tell which document is ignored so it's not gonna be a really actionable message